test-support-apacheds2/src/main/java/org/springframework/ldap2/test/EmbeddedLdapServer.java

```diff
@@ -0,0 +1,91 @@
+/*
```

Comment:
I'm not the biggest fan of the package name being ldap2. To me this signals that it is Spring LDAP v2 versus indicating that Apache DS is version 2. I'm guessing you did this because you wanted to avoid class name collisions and the current packaging does not have a distinct package for apacheds. 

Perhaps creating a new package named org.springframework.ldap.test.apacheds2 makes sense? I realize this seems strange when comparing to the original package but I think it is an improvement. What are your thoughts?